### ISSUE TITLE
More IDs

### ISSUE THREAD

**Issue by iamc24** [Texthead/Skylander-IDs#1]
[https://pastebin.com/jgCHBzaK](https://pastebin.com/jgCHBzaK)

All read using my own figures with Skylanders GUI Tool dumps
All figures confirmed working and identifying correctly in SuperChargers Wii U

**Comment by Texthead** [comment]
Only getting around to this now, my apologies.

Variant IDs have all been useful and seemingly correct, however the majority of the CharacterIDs are not the correct ones from figures; for instance Sheep Wreck Island and Jet Stream both apparently share the same ToyCode? I can verify that this isn't the case, so I can't take all this info as directly legitimate, however I do already have the correct CharacterIDs for all the aforementioned figures, so all is well

Regardless, thank you

**Comment by iamc24** [comment]
I'm glad that the variant IDs helped at least. I see what happened with the character IDs though. I have a separate list with all the character and variant IDs of my figures for identifying my dump files if something gets miss named or otherwise messed up somehow. The character IDs got mixed up when I was copying from my list to make and format the list I linked. I made sure to double-check the variant IDs after converting them to decimal to make sure they were equivalent because of how important they are for the games with stricter figure identification. However, I never double-checked the character IDs, so they got misaligned when sorting and/or converted incorrectly.
